File: LinkedList.java
/* Linked List in Java */

class SingleLinkedList {
    public static void main(String args[]) {
        Node head = new Node(6);
        head.next = new Node(9);
        head.next.next = new Node(7);
        head.printLinkedList(head);
    }
}
class Node {
    int data;
    Node next;
    Node(int data) {
        this.data = data;
        this.next = null;
    }
    void printLinkedList(Node head) {
        while (head != null) {
            System.out.println(head.data);
            head = head.next;
        }
    }
}

File: LinkedList.cpp
/* LINKED LIST IN CPP */
#include <iostream>
using namespace std;
class Node
{
public:
    int data;
    Node *next;
    Node(int data)
    {
        this->data = data;
        this->next = nullptr;
    }
    void printLinkedList(Node *head)
    {
        while (head != nullptr)
        {
            cout << head->data << endl;
            head = head->next;
        }
    }
};
int main()
{
    Node *head = new Node(6);
    head->next = new Node(9);
    head->next->next = new Node(7);
    head->printLinkedList(head);
    return 0;
}